src/containers/layouts/articles/article3/article3.container.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { NavigationScreenProps } from 'react-navigation';
+import { Article } from '@src/core/model';
+import { articles } from '@src/core/data/article';
+import { Article3 } from './article3.component';
+
+interface State {
+  article: Article;
+  comments: string[];
+  currentComment: string;
+}
+
+export class Article3Container extends React.Component<NavigationScreenProps, State> {
+
+  public state: State = {
+    article: articles[0],
+    comments: [
```

Comment:
I think it would be better if we declare comments property inside an article model

Reply:
Will be refactored in the next PR.